lhotse/cut.py

```diff
@@ -2792,7 +2799,7 @@ def load_audio(self, mixed: bool = True) -> Optional[np.ndarray]:
                 )
             except NonPositiveEnergyError as e:
                 logging.warning(
-                    str(e) + f' MonoCut with id "{track.cut.id}" will not be mixed in.'
+                    f'{e} MonoCut with id "{track.cut.id}" will not be mixed in.'
```

Comment:
@pzelasko I believe the try-except block is not needed here since the energy is checked in the constructor, or do I miss something?